package proto_ff;

// calm 自定义的fieldoption都在该文件中
import "yd_fieldoptions.proto";

message emailSender
{
  optional string email = 1;
  optional string title = 2;
  optional string msg = 3;
}
